docs/azure_ad_config_guide.rst

```diff
@@ -135,6 +135,6 @@ Add this path to your projects ``urls.py`` file.
 
     urlpatterns = [
         ...
-        path('oauth2/', include('django_auth_adfs.drf-urls')),
+        path('oauth2/', include('django_auth_adfs.urls')),
```

Comment:
Hi, this is a good catch. I think we should have both, just to document the options. The reason to have `drf-urls` is to override the standard Django Rest Framework login pages in your main `urls.py`.

```suggestion
        path('oauth2/', include('django_auth_adfs.urls')),
        path('oauth2/', include('django_auth_adfs.drf-urls')),
```

Reply:
Keep in mind that not everyone uses `django-rest-framework` in their project. It would be great to add a comment in the guide about which path must be included in the `urls.py` file depending on your app configuration.

Reply:
That's a good point. The documentation for DRF will tell you to add this specifically, and you have to read through that documentation if you use DRF regardless of which AD setup you use. I'll approve yours. Thank you.